spec/numberformat.html

```diff
@@ -1304,7 +1311,7 @@ <h1>
           1. Set _m_ to the string-concatenation of *"0."*, -(_e_ + 1) occurrences of the code unit 0x0030 (DIGIT ZERO), and _m_.
           1. Let _int_ be 1.
         1. If _m_ contains the code unit 0x002E (FULL STOP) and _maxPrecision_ > _minPrecision_, then
-          1. Let _cut_ be _maxPrecision_ - _minPrecision_.
+          1. Let _cut_ be _maxPrecision_ - max(_fractionDigits_, _minPrecision_).
```

Comment:
It seems wrong to compare fraction digits and significant digits

Reply:
You're right; the value needs to be offset by the integer digits count here.
```suggestion
          1. Let _integerDigits_ be the count of code units in _m_ preceding the code unit 0x002E (FULL STOP).
          1. Let _cut_ be _maxPrecision_ - max(_integerDigits_ + _fractionDigits_, _minPrecision_).
```